devlib/instrument/gem5power.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, target, power_sites):
             self.add_channel(field, 'power')
         self.target.gem5stats.book_roi(self.roi_label)
         self.sample_period_ns = 10000000
+        self.sample_rate_hz = 1 / (self.sample_period_ns * 10**-9)
```

Comment:
Hmmm yes that's an issue unfortunately ... In short, `sim_seconds` is gem5-related and cannot be changed. In more detail, with the current implementation, the channel names (including the `power_sites`) of the `Gem5PowerInstrument` must match specific fields of the statistics log file of gem5. And as you probably guessed, the field that gives the time elapsed since the last sample is `sim_seconds` ... So if this is absolutely required to change the `sim_seconds` to `timestamp`, we probably need deeper modifications of `Gem5PowerInstrument` to decorelate the channel names from the fields of the stats file. Or we can hack something for the specific case of `sim_seconds` ...